examples/alarmBot/abilities.ts

```diff
@@ -57,22 +58,18 @@ export const abilities = [
             isValid: true
           }
         },
-        onFill: (value) => `ok! time is set to ${value}.`
+        onFill: (s, value) => `ok! time is set to ${value}.`
       }
     ],
-    onComplete: (convoState, submittedData) => {
-      return new Promise((resolve, reject) => {
+    onComplete: async ({read}, submittedData) => {
         const value = submittedData
+        const convoState = await read()
         const alarms = convoState.alarms || []
         convoState.alarms = [
```

Comment:
Do we want to utilize `save` pattern instead of saving back to the state directly?

Reply:
@KSLHacks Added save to all the onComplete functions in all example bots